src/DependencyInjection/SearchRequirementsPass.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Algolia\SearchBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\PriorityTaggedServiceTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+
+class SearchRequirementsPass implements CompilerPassInterface
+{
+    use PriorityTaggedServiceTrait;
+
+    public function process(ContainerBuilder $container)
+    {
+        $servicesNeeded = ['serializer'];
+
+        foreach ($servicesNeeded as $service) {
+            if (! $container->has($service)) {
```

Comment:
extra space that should be removed after the `!`

Reply:
I need to setup php-cs-fixer with Symfony standards ^^